apps/experiments/views/experiment.py

```diff
@@ -1488,3 +1493,21 @@ def get_release_status_badge(request, team_slug: str, experiment_id: int):
     experiment = get_object_or_404(Experiment, id=experiment_id, team=request.team)
     context = {"has_changes": experiment.compare_with_latest(), "experiment": experiment}
     return render(request, "experiments/components/unreleased_badge.html", context)
+
+
+def migrate_experiment_view(request, team_slug, experiment_id):
+    experiment = get_object_or_404(Experiment, id=experiment_id, team__slug=team_slug)
+    failed_url = reverse(
+        "experiments:single_experiment_home",
+        kwargs={"team_slug": team_slug, "experiment_id": experiment_id},
+    )
+    try:
+        task = migrate_experiment_to_pipeline_task.delay(experiment_id)
+        task.get(timeout=60)
```

Comment:
If we're going to wait for the task to complete then we might as well just run it in the request thread instead of using Celery. Considering that it's just creating a new pipeline from existing data it should be fast so I don't see an issue with doing it synchronously.